base-containers/base/inginious_container_api/run_student.py

```diff
@@ -37,13 +40,22 @@ def run_student(cmd, container=None,
                                     this function can itself be called with a signal value that will immediately be sent
                                     to the remote process. See the run_student script command for an example, or
                                     the hack_signals function below.
+    :param ssh: If set to True, it starts an ssh server for the student after the command finished.
+    :param run_as_root: If set to True, it tries to execute the command as root (for ssh, it accepts connection as root).
+                        Default is False. This is a Beta feature and should not be used yet.
```

Comment:
Add a warning saying that it depends on the runtime?

Reply:
Well actually it should not depend on the runtime.
Well right now the entire "run_student" thing is unavailable for kata indeed....
But when run_student will be available for kata, the "run_as_root" parameter has nothing to do with the docker or kata runtime being selected for the task (in other words, a docker grading_container should be able to start a kata root student_container)

In other words, the entire parameter "run_as_root" is useful for now indeed run_student strats by exiting if run_as_root is set to true but it will have sense in a near future.